src/app/App.js

```diff
@@ -17,19 +17,40 @@ export const App = ({options}) => {
    //add event listener to the select mode menu button 
    const help=document.querySelector('#helpScreen')
    const style = getComputedStyle(help);
+   
+   const changeColorOfClickedButton = (querySelector) => {
+
+      const whoIsThatPokemonOption = document.querySelector('#whatItLooksLikeOption')
+      const whatItLooksLikeOption = document.querySelector('#guessTheTypeOption')
+      const guessTheTypeOption = document.querySelector('#whoIsThatPokemonOption')
+
+      whoIsThatPokemonOption.style.backgroundColor="#3762AC"
+      whoIsThatPokemonOption.style.color="#FFCB05"
```

Comment:
You can extract those strings to some variables with meaningful name.

Remember: when you create repetition in code, ask yourself: "How many places I would have to change in order to change this functionality?" - the best answer is **one**.

Now, if you would want to change `#3762AC` color you would need to change 5 places: 4 places in code here and 1 in css.

The best solution for this case would be to have 1 place to change in js and 1 place to change in CSS